examples/svm/plot_svm_anova.py

```diff
@@ -20,16 +20,17 @@
 from sklearn.svm import SVC
 
 
-# #############################################################################
+# %%
 # Import some data to play with
+# ------------------------------------------------
 X, y = load_iris(return_X_y=True)
 # Add non-informative features
 np.random.seed(0)
 X = np.hstack((X, 2 * np.random.random((X.shape[0], 36))))
 
-# #############################################################################
-# Create a feature-selection transform, a scaler and an instance of SVM that we
-# combine together to have a full-blown estimator
+# %%
+# Create a feature-selection transform, a scaler and an instance of SVM that we combine together to have a full-blown estimator
```

Comment:
This line is too long. Try to follow the [PEP 8 convention](https://www.python.org/dev/peps/pep-0008/), i.e. try to limit all lines to a maximum of 79 characters.